src/masternode-payments.cpp

```diff
@@ -777,16 +777,10 @@ std::string CMasternodePayments::ToString() const
     return info.str();
 }
 
-bool CMasternodePayments::IsEnoughData(int nMnCount) {
-    if(GetBlockCount() > nMnCount * nStorageCoeff && GetBlockCount() > nMinBlocksToStore)
```

Comment:
Thinking more of it, I guess I shouldn't mix these two changes into one commit, so I'm bringing GetBlockCount back now.